src/packages/cli/src/Init.ts

```diff
@@ -143,6 +167,11 @@ export class Init implements Command {
       url = args['--url']
     }
 
+    if (args['--provider'] && !args['--url']) {
```

Comment:
```suggestion
    } else if (args['--provider']) {
```
Minor suggestion: I think this could be better